SlothfulSheep/bangazon-api-sprint2
# Ability to Choose Active customer from Prompt

# Description

This pull request completes all requirements for issue #2. Users are now able to choose an active customer from the command line. We also introduced a test for the getAll customers function.


## Type of change


# How Has This Been Tested?

To test the command line run the following commands:
```
npm start
```
- Select number 2 from the command line.
- You should see a list of all customers.
- Enter a customer ID number and hit enter.
- After selecting a customer you should be directed back to the initial Welcome prompt. There will be a message within the prompt that indicates which customer is now listed as your active customer.


**RUNNING TESTS**

To initiate the testing run the following command:
```
npm test
```
- There should be 8 test that pass.


# Checklist:
